geode-apis-compatible-with-redis/src/distributedTest/java/org/apache/geode/redis/session/springRedisTestApplication/config/DUnitSocketAddressResolver.java

```diff
@@ -1,57 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
- * agreements. See the NOTICE file distributed with this work for additional information regarding
- * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance with the License. You may obtain a
- * copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package org.apache.geode.redis.session.springRedisTestApplication.config;
-
-import java.net.InetAddress;
-import java.net.InetSocketAddress;
-
-import io.lettuce.core.RedisURI;
-import io.lettuce.core.resource.DnsResolver;
-import io.lettuce.core.resource.SocketAddressResolver;
-import org.apache.logging.log4j.Logger;
-
-import org.apache.geode.logging.internal.log4j.api.LogService;
-
-public class DUnitSocketAddressResolver extends SocketAddressResolver {
```

Comment:
Doesn't seem like we need this any longer, but I wanted to make sure it was intentionally delted.

Reply:
Yup. This was a hacky way to get non-cluster aware clients to work with our older hybrid cluster model. Since we've moved to a pure cluster mode we should be able to rely on clients like `jedis` and `lettuce` to be able to automatically adjust to topology changes.